tests/phpunit/tests/rest-api/rest-posts-controller.php

```diff
@@ -1517,6 +1517,47 @@ public function test_get_items_not_sticky_with_exclude_no_sticky_posts() {
 		$this->assertPostsWhere( " AND {posts}.ID NOT IN ($id3) AND {posts}.post_type = 'post' AND (({posts}.post_status = 'publish'))" );
 	}
 
+	/**
+	 * @ticket 55592
+	 * @covers WP_REST_Posts_Controller::get_items()
+	 */
+	public function test_get_items_with_featured_media() {
```

Comment:
This test is failing in a bunch of places, are you able to take a look as to why?

Reply:
@spacedmonkey maybe is the order of $attachment_ids.